diff -Naur old/Time/DateStrings.cpp new/Time/DateStrings.cpp
--- old/Time/DateStrings.cpp	2010-03-06 11:09:22.000000000 +0100
+++ new/Time/DateStrings.cpp	2014-08-25 09:42:01.523638843 +0200
@@ -15,38 +15,38 @@
 
 static char buffer[dt_MAX_STRING_LEN+1];  // must be big enough for longest string and the terminating null
 
-char monthStr1[] PROGMEM = "January";
-char monthStr2[] PROGMEM = "February";
-char monthStr3[] PROGMEM = "March";
-char monthStr4[] PROGMEM = "April";
-char monthStr5[] PROGMEM = "May";
-char monthStr6[] PROGMEM = "June";
-char monthStr7[] PROGMEM = "July";
-char monthStr8[] PROGMEM = "August";
-char monthStr9[] PROGMEM = "September";
-char monthStr10[] PROGMEM = "October";
-char monthStr11[] PROGMEM = "November";
-char monthStr12[] PROGMEM = "December";
+const char monthStr1[] PROGMEM = "January";
+const char monthStr2[] PROGMEM = "February";
+const char monthStr3[] PROGMEM = "March";
+const char monthStr4[] PROGMEM = "April";
+const char monthStr5[] PROGMEM = "May";
+const char monthStr6[] PROGMEM = "June";
+const char monthStr7[] PROGMEM = "July";
+const char monthStr8[] PROGMEM = "August";
+const char monthStr9[] PROGMEM = "September";
+const char monthStr10[] PROGMEM = "October";
+const char monthStr11[] PROGMEM = "November";
+const char monthStr12[] PROGMEM = "December";
 
-PGM_P monthNames_P[] PROGMEM = 
+PGM_P const monthNames_P[] PROGMEM = 
 {
     "",monthStr1,monthStr2,monthStr3,monthStr4,monthStr5,monthStr6,
 	monthStr7,monthStr8,monthStr9,monthStr10,monthStr11,monthStr12
 };
 
-char monthShortNames_P[] PROGMEM = "ErrJanFebMarAprMayJunJulAugSepOctNovDec";
+const char monthShortNames_P[] PROGMEM = "ErrJanFebMarAprMayJunJulAugSepOctNovDec";
 
-char dayStr0[] PROGMEM = "Err";
-char dayStr1[] PROGMEM = "Sunday";
-char dayStr2[] PROGMEM = "Monday";
-char dayStr3[] PROGMEM = "Tuesday";
-char dayStr4[] PROGMEM = "Wednesday";
-char dayStr5[] PROGMEM = "Thursday";
-char dayStr6[] PROGMEM = "Friday";
-char dayStr7[] PROGMEM = "Saturday";
+const char dayStr0[] PROGMEM = "Err";
+const char dayStr1[] PROGMEM = "Sunday";
+const char dayStr2[] PROGMEM = "Monday";
+const char dayStr3[] PROGMEM = "Tuesday";
+const char dayStr4[] PROGMEM = "Wednesday";
+const char dayStr5[] PROGMEM = "Thursday";
+const char dayStr6[] PROGMEM = "Friday";
+const char dayStr7[] PROGMEM = "Saturday";
 
-PGM_P dayNames_P[] PROGMEM = { dayStr0,dayStr1,dayStr2,dayStr3,dayStr4,dayStr5,dayStr6,dayStr7};
-char dayShortNames_P[] PROGMEM = "ErrSunMonTueWedThrFriSat";
+PGM_P const dayNames_P[] PROGMEM = { dayStr0,dayStr1,dayStr2,dayStr3,dayStr4,dayStr5,dayStr6,dayStr7};
+const char dayShortNames_P[] PROGMEM = "ErrSunMonTueWedThrFriSat";
 
 /* functions to return date strings */
 
